pkg/cmd/roachprod/main.go

```diff
@@ -794,15 +796,21 @@ stage it on the cluster. It will then perform a rolling upgrade of the cluster,
 one node at a time, to the new version.
 
 Currently available application options are:
-  %s`, strings.TrimSpace(cockroachApp+releaseApp+customizedApp)),
+  %s`, strings.TrimSpace(cockroachApp+releaseApp+customizedApp+localApp)),
 	Args: cobra.RangeArgs(2, 3),
 	Run: wrap(func(cmd *cobra.Command, args []string) error {
 		versionArg := ""
-		if len(args) == 3 {
+		binaryPath := ""
+		if args[1] == "local" {
+			if len(args) < 3 {
+				return errors.New("local application requires a binary path: deploy <cluster> local <binary-path>")
```

Comment:
Nit: "...requires a path to the binary" and `binary-path` -> `pathToBinary`.

Reply:
Done.